docs/sql-ref-syntax-dml-load.md

```diff
@@ -19,4 +19,63 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`LOAD DATA` loads data from a directoy or a file into a table or into a partition in the table. A partition spec should be specified whenever the target table is partitioned. The `LOAD DATA` statement can only be used with tables created using the Hive format.
+
+### Syntax
+{% highlight sql %}
+LOAD DATA [ LOCAL ] INPATH path [ OVERWRITE ] INTO TABLE table_name
+  [ PARTITION ( partition_col_name [ = partition_col_val ] [ , ... ] ) ]
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>path</em></code></dt>
+  <dd>Path of the file system.</dd>
```

Comment:
@huaxingao I was looking up the hive documentation. They mention about load deleting the data from source directory i.e they do a move operation. I think this is a important thing to say if that is the behaviour.

Reply:
@huaxingao Also seems like this path can be both an absolute path or relative path. Do we need to mention that here ?

Reply:
@dilipbiswal I checked, actually the data in the source directory is not deleted.